generic/tests/netperf.py

```diff
@@ -93,6 +93,7 @@ def env_setup(session, ip, user, port, password):
         error.context("Setup env for %s" % ip)
         ssh_cmd(session, "iptables -F", ignore_status=True)
         ssh_cmd(session, "service iptables stop", ignore_status=True)
```

Comment:
Yes,  service module could choose which command should be used. but it's not accurate by above code since env_setup was call three times with  "prepare_list = set([server_ctl, client, host]) " , two with ShellSession (one is guest shell, one is external host shell)and one with localhost, so there only guest_run is not suitable. 

And when "i=localhost", still hit error below. 
 2017-07-10 03:20:27,789 stacktrace       L0045 ERROR|     firewalld = service.Factory.create_service("firewalld", run=guest_run.run)
2017-07-10 03:20:27,789 stacktrace       L0045 ERROR|   File "/home/staf-kvm-devel/workspace/avocado-vt/virttest/staging/service.py", line 872, in create_service
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR|     return Factory.create_specific_service(service_name, run)
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR|   File "/home/staf-kvm-devel/workspace/avocado-vt/virttest/staging/service.py", line 854, in create_specific_service
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR|     helper = Factory.FactoryHelper(run)
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR|   File "/home/staf-kvm-devel/workspace/avocado-vt/virttest/staging/service.py", line 720, in __init__
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR|     result = run("true")
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR|   File "/home/staf-kvm-devel/workspace/avocado-vt/virttest/remote.py", line 1238, in run
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR|     status, _ = self.session.cmd_status_output(command, timeout=timeout)
2017-07-10 03:20:27,790 stacktrace       L0045 ERROR| AttributeError: 'str' object has no attribute 'cmd_status_output'

So, I still suggest my commit. Thanks~~